lua/starfall/libs_sh/vehicles.lua

```diff
@@ -167,6 +225,45 @@ if SERVER then
 		UseEnableVehicles:setEnabled(veh, enabled, key)
 	end
 
+	--- Toggles the vehicles entry/exit camera animation
+	-- @param boolean Enabled
+	-- @server
+	function vehicle_methods:setVehicleEntryAnim(enabled)
```

Comment:
Did you figure out if this was broken or not? Should probably remove it if it isn't functional in glua